src/index.js

```diff
@@ -21,3 +21,4 @@ export { default as SnapPageLogin } from './SnapPage/SnapPageLogin'
 export { default as SnapPagePurchase } from './SnapPage/SnapPagePurchase'
 export { default as SnapPageSummary } from './SnapPage/SnapPageSummary'
 export { default as SnapPageTags } from './SnapPage/SnapPageTags'
+export { default as StoreSections } from './Store/StoreSections'
```

Comment:
I think these can just be:
```
export StoreSections from from './Store/StoreSections';
```

but maybe there's a reason for doing it this way?